amplify/backend/function/jcProcessDMAlerts/src/index.js

```diff
@@ -203,6 +251,7 @@ async function emailRouter(html, text, fromInfo, messageRoomID, recipientInfo) {
 
 async function Execute(event) {
   await asyncForEach(event.Records, async (record) => {
+    console.log(record)
```

Comment:
It might be useful to console.log the messageRoomID, id, and userID of the new record.